pkg/agent/proxy/proxier.go

```diff
@@ -159,7 +160,7 @@ func (p *proxier) removeStaleServices() {
 			}
 		}
 		// Remove LoadBalancer flows and configurations.
-		if len(svcInfo.LoadBalancerIPStrings()) > 0 {
+		if p.proxyExternalIPs && len(svcInfo.LoadBalancerIPStrings()) > 0 {
```

Comment:
I just found when we talked about externalIP, we actually referred to loadbalancerIPs, while there is another field in Service's Spec called `ExternalIPs`, which Antrea doesn't proxy at the moment but kube-proxy does. We may need to support it in the future. I don't think we need to have a toggle for it as the spec says "externalIPs is a list of IP addresses for which nodes in the cluster will also accept traffic for this service". But to avoid confusion, should we call the configuration for loadbalancer IPs `proxyLoadBalancerIPs`?

Reply:
I actually opened an issue a while back about supporting `ExternalIPs` for ClusterIP Services: https://github.com/antrea-io/antrea/issues/1627

Originally I thought that the `proxyExternalIPs` configuration would eventually apply to both (LoadBalancerIPs and ExternalIPs) but thinking about it some more I don't think it makes sense. I'll work on renaming the configuration parameter as you suggested.